### DISCUSSION TITLE
[FR] Add list for water meters

### DISCUSSION THREAD

**Discussion by el97** [huizebruin/s0tool#58] [General] [answered]
First of I want to say great work with this project! I think that this will come in handy for many users that want to track water usage and so on.
I would like to see a list for water meters support this function. That will make it easier for others to see if they can use this for tracking water usage and then buy the needed components. The list can be implemented if people share their experience and then write if it works or not for their water meter.

**Comment by huizebruin** [comment, marked as answer]
I made a list here [watermeters list](https://github.com/huizebruin/s0tool/discussions/57)
